### PR TITLE
[WNMGDS-321] Update class references to use .classname format

### PR DESCRIPTION
**Pull request by line47**
Updated all places where a classname is used to use the `.CLASSNAME` format that inludes a `.`

### CODE REVIEW

File: packages/design-system-docs/src/scripts/components/HtmlExample.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ class HtmlExample extends React.PureComponent {
     return (
       <div className="c-markup__header">
         <h4 className="ds-u-font-size--h5 ds-u-margin-bottom--0">
-          Modifier: <code>{this.name()}</code>
+          Modifier: <code>.{this.name()}</code>
```
Comment (line47) [inline packages/design-system-docs/src/scripts/components/HtmlExample.jsx:36]: Updated the HTML example so the `.` is always included.

**Comment by line47** [comment]
Thanks for the review @nichia I've updated the PR 👍

**Comment by nichia** [comment]
Have checked the PR and it's good to go.

File: packages/design-system-docs/src/example.js [hunk 2]
```diff
@@ -31,14 +31,14 @@ function setModalExamples() {
   const dialogHeading = document.getElementById('dialog-heading');
   const dialogWrap = document.getElementById('dialog-wrap');
 
-  document.querySelectorAll('.dialog-open-btn').forEach((btn) => {
+  document.querySelectorAll('.dialog-open-btn').forEach(btn => {
```
Comment (bernardwang) [inline packages/design-system-docs/src/example.js:34]: @line47 A lot of these style changes should not have been included in this PR, as they go against our new linting rules. I'm not sure exactly why they are being included here (maybe your local dev env needs to be updated or reconfigured), but if you run `yarn lint` these changes should be undone.

Comment (line47) [reply packages/design-system-docs/src/example.js:34]: Hmmm yeah that is odd, I'll take a look. Thanks!

Comment (line47) [reply packages/design-system-docs/src/example.js:34]: Looks like these were introduced when I merged master into this branch 
https://github.com/CMSgov/design-system/pull/677/commits/61ebb45f4a799301ffe934ae0d4bdfd5266090b1

**Pull request merged**